Userland/Libraries/LibGfx/ImageFormats/GIFWriter.cpp

```diff
@@ -21,15 +21,15 @@ ErrorOr<void> write_header(Stream& stream)
     return {};
 }
 
-ErrorOr<void> write_logical_descriptor(BigEndianOutputBitStream& stream, Bitmap const& bitmap)
+ErrorOr<void> write_logical_descriptor(BigEndianOutputBitStream& stream, IntSize dimensions)
```

Comment:
nit: s/dimensions/size/

Reply:
I wrote `dimensions` because that's what you used here lol:
`WebPAnimationWriter(SeekableStream& stream, IntSize dimensions, u8 original_vp8x_flags)`